src/components/Agents/List/AgentsTableColumns.tsx

```diff
@@ -0,0 +1,75 @@
+import { ColumnDef } from "@tanstack/react-table";
+import { Agent } from "../AgentPage";
+import { DateCell } from "../../ConfigViewer/columns";
+import { User } from "../../../api/services/users";
+import { Avatar } from "../../Avatar";
+
+export const agentsTableColumns: ColumnDef<Agent>[] = [
+  {
+    header: "Name",
+    accessorKey: "name",
+    minSize: 150,
+    enableSorting: true
+  },
+  {
+    header: "Hostname",
+    accessorKey: "hostname",
+    minSize: 150,
+    enableSorting: true
+  },
+  {
+    header: "IP Address",
+    accessorKey: "ip",
+    minSize: 150,
+    enableSorting: true
+  },
+  {
+    header: "Person",
+    accessorKey: "person_id",
+    minSize: 150,
+    cell: ({ row }) => {
+      const person = row?.getValue<User>("person_id");
+      return <Avatar user={person} />;
+    }
+  },
+  {
+    header: "Version",
+    accessorKey: "version",
+    minSize: 80,
+    enableSorting: true
+  },
+
+  {
+    header: "Username",
+    accessorKey: "username",
+    minSize: 150,
+    enableSorting: true
+  },
+  {
+    header: "TLS",
+    accessorKey: "tls",
+    minSize: 80,
+    enableSorting: true
+  },
```

Comment:
We only need name, created/updated_at, created_by, labels, and the counts as per : https://github.com/flanksource/duty/blob/main/views/022_agents.sql#L1